core/controllers/acl_decorators.py

```diff
@@ -1366,9 +1364,7 @@ def test_primary_admin(
             raise self.NotLoggedInException
 
         email = user_services.get_email_from_user_id(self.user_id)
-        if email != platform_parameter_services.get_platform_parameter_value(
-            platform_parameter_list.ParamName.SYSTEM_EMAIL_ADDRESS.value
-        ):
+        if email != feconf.SYSTEM_EMAIL_ADDRESS:
```

Comment:
Just a note -- the changes here seem correct but I'm confused, do we already have the platform param value being used in prod? That seems like a problem -- we should be reverting the PR that had the original change, right?

I found https://github.com/oppia/oppia/commit/1723f621ca1d14f2c35080163fcf2d22a53f4ee7 but did it get merged? I would have expected the changes on the **left** panel to be showing "feconf.SYSTEM_EMAIL_ADDRESS" (and this file to not even show any changes during the PR review).

/cc @kevintab95

Reply:
Thanks Sean, that makes sense. I have separated the revert since it made this PR confusing. I did use the platform params in prod defaulting them to feconf values so that indirectly feconf values from release update are being used. However, after our discussion on the testing doc, it made sense to keep develop in safe state. The revert is in here: https://github.com/oppia/oppia/pull/21264. Would it be possible to merge that one sooner as it can help me clear up the other PRs and proceed faster. Thank you!